### PR TITLE
Branch fix bugs in autoschedule

### PR DESCRIPTION
**Pull request by oscarsu97**

### CODE REVIEW

File: src/main/java/seedu/planner/logic/commands/AutoScheduleCommand.java [hunk 1]
```diff
@@ -179,13 +186,15 @@ public CommandResult execute(Model model) throws CommandException {
                         LocalTime startTimeOfNextActivity = timeSchedule.get(nextTimingIndex);
                         LocalTime currentActivityEndTime = currentTiming.plusMinutes(duration);
                         if (startTimeOfNextActivity.compareTo(currentActivityEndTime) >= 0) {
+                            System.out.println(currentTiming);
```
Comment (1nefootstep) [inline src/main/java/seedu/planner/logic/commands/AutoScheduleCommand.java]: remove these debug statements. Also, you should try to debug with the debugger, it's far more flexible than print statements

File: src/main/java/seedu/planner/logic/commands/AutoScheduleCommand.java [hunk 2]
```diff
@@ -179,13 +186,15 @@ public CommandResult execute(Model model) throws CommandException {
...
                             isScheduled = true;
                             activitiesForTheDay.add(activityToSchedule(timeSchedule.get(i),
                                     duration, activityWithCount.getActivity()));
                             //the next activity will be given a start time,
                             // if the timing is not the next in line
                             //Eg. 1000 null 1300 -> becomes 1000 1000+30min  1300
                             if (nextTimingIndex != i + 1) {
+                                System.out.println("here");
```
Comment (1nefootstep) [inline src/main/java/seedu/planner/logic/commands/AutoScheduleCommand.java]: remove

File: src/main/java/seedu/planner/logic/commands/AutoScheduleCommand.java [hunk 3]
```diff
@@ -163,10 +166,14 @@ public CommandResult execute(Model model) throws CommandException {
                     int nextTimingIndex = -1;
                     LocalTime currentTiming = timeSchedule.get(i);
                     for (int k = i + 1; k < timeSchedule.size(); k++) {
+                        System.out.println("timeSchedule: " + timeSchedule.get(k));
```
Comment (1nefootstep) [inline src/main/java/seedu/planner/logic/commands/AutoScheduleCommand.java]: remove

File: src/main/java/seedu/planner/logic/commands/AutoScheduleCommand.java [hunk 4]
```diff
@@ -100,6 +100,9 @@ public CommandResult execute(Model model) throws CommandException {
             List<Activity> lastShownActivities = model.getFilteredActivityList();
             List<Activity> filteredActivitiesByLocation = lastShownActivities;
             List<LocalTime> timeSchedule = fillTimeSchedule(draftSchedule);
+            for (LocalTime localTime : timeSchedule) {
+                System.out.println("time:" + localTime);
```
Comment (1nefootstep) [inline src/main/java/seedu/planner/logic/commands/AutoScheduleCommand.java]: remove

File: src/main/java/seedu/planner/logic/commands/AutoScheduleCommand.java [hunk 5]
```diff
@@ -163,10 +166,14 @@ public CommandResult execute(Model model) throws CommandException {
                     int nextTimingIndex = -1;
                     LocalTime currentTiming = timeSchedule.get(i);
                     for (int k = i + 1; k < timeSchedule.size(); k++) {
+                        System.out.println("timeSchedule: " + timeSchedule.get(k));
                         if (timeSchedule.get(k) != null) {
+                            System.out.println("k: " + k);
```
Comment (1nefootstep) [inline src/main/java/seedu/planner/logic/commands/AutoScheduleCommand.java]: remove

File: src/main/java/seedu/planner/logic/commands/AutoScheduleCommand.java [hunk 6]
```diff
@@ -151,20 +152,28 @@ public CommandResult execute(Model model) throws CommandException {
                 //Checks if duration exceed the next timing if any
                 for (ActivityWithCount activityWithCount : activitiesWithCount) {
                     int duration = activityWithCount.getActivity().getDuration().value;
+                    LocalTime currentTiming = timeSchedule.get(i);
+                    LocalTime currentActivityEndTime = currentTiming.plusMinutes(duration);
+                    int currentTime = currentTiming.getHour() * 1000 + currentTiming.getMinute();
+                    int nextTime = currentActivityEndTime.getHour() * 1000 + currentActivityEndTime.getMinute();
```
Comment (1nefootstep) [inline src/main/java/seedu/planner/logic/commands/AutoScheduleCommand.java]: these should have been in LocalTime objects

**Comment by oscarsu97** [comment]
Closes #143

**Pull request merged**